src/node_options.cc

```diff
@@ -849,8 +850,8 @@ void GetOptions(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
   if (!env->has_run_bootstrapping_code()) {
     // No code because this is an assertion.
```

Comment:
This is the comment stating why we're not doing this :) I'd leave this as is, we don't need error codes if error can't be triggered through public API

Reply:
booo... lol... ok, yeah that makes sense